app/commands/payments/stripe/payment/create.rb

```diff
@@ -14,14 +14,21 @@ def initialize
   initialize_with :user, :stripe_data, subscription: nil
 
   def call
-    Donations::Payment::Create.(user, :stripe, external_id, amount_in_cents, external_receipt_url, subscription:)
+    Payments::Payment::Create.(user, :stripe, product, external_id, amount_in_cents, external_receipt_url, subscription:)
   end
 
   private
   def external_id = stripe_data.id
   def external_receipt_url = stripe_data.charges.first.receipt_url
   def amount_in_cents = stripe_data.amount
 
+  def product
+    return subscription.product if subscription
+
+    # Premium payments are always linked to a subscription, even one-off payments
```

Comment:
```suggestion
    # Premium payments are always linked to a subscription
```

As we no-longer have one-off product payments?